Cargo.lock

```diff
@@ -3100,11 +3125,11 @@ dependencies = [
 
 [[package]]
 name = "time"
-version = "0.3.11"
+version = "0.3.14"
```

Comment:
Checking if dependencies are up-to-date fails

```
error: package `time v0.3.14` cannot be built because it requires rustc 1.59.0 or newer, while the currently active rustc version is 1.58.1
Error: The process '/home/runner/.cargo/bin/cargo' failed with exit code 101
```